PWGGA/GammaConv/AliAnalysisTaskEtaToPiPlPiMiGamma.cxx

```diff
@@ -875,8 +875,8 @@ void AliAnalysisTaskEtaToPiPlPiMiGamma::ProcessPhotonCandidates(){
 void AliAnalysisTaskEtaToPiPlPiMiGamma::ProcessTruePhotonCandidates(AliAODConversionPhoton *TruePhotonCandidate)
 {
 	// Process True Photons
-    TParticle *posDaughter = TruePhotonCandidate->GetPositiveMCDaughter(fMCEvent);
-    TParticle *negDaughter = TruePhotonCandidate->GetNegativeMCDaughter(fMCEvent);
+    AliVParticle *posDaughter = TruePhotonCandidate->GetPositiveMCDaughter(fMCEvent);
```

Comment:
I think there is no particular reasoning for that, but since AliMCParticle inherits from AliVParticle I think it is fine right.
Anyway I can change that if you prefer. It would be more consistent.